src/PluginForm/Bluesnap/HostedPaymentFieldsPaymentMethodAddForm.php

```diff
@@ -47,7 +48,8 @@ public function __construct(
     RouteMatchInterface $route_match,
     EntityTypeManagerInterface $entity_type_manager,
     LoggerInterface $logger,
-    ClientFactory $client_factory
+    ClientFactory $client_factory,
+    FraudSession $fraud_session
```

Comment:
**CHANGE REQUEST**

Class property and function argument declaration.

Reply:
Added